src/main/java/org/sagebionetworks/web/client/widget/table/TableListWidget.java

```diff
@@ -72,6 +74,11 @@ public void configure(EntityBundle parentBundle, List<EntityType> typesToShow) {
 		isInitializing = true;
 		this.parentBundle = parentBundle;
 		this.typesToShow = typesToShow;
+		if (typesToShow.size() == 1 && typesToShow.contains(EntityType.dataset)) {
+			this.view.setTableType(TableType.dataset);
+		} else {
+			this.view.setTableType(TableType.table);
+		}
```

Comment:
The only two cases are the datasets tab and tables tab so this is super specific and not very generalizable, but I didn't want to spend more time handling copy for cases we don't have, and might not ever have.